server/routes/take-quiz.js

```diff
@@ -0,0 +1,51 @@
+const express = require('express')
+const router = express.Router()
+const fetch = require("../database/scripts/take-quiz/utilities");
+
+router.get('/', (req, res) => {
+  res.send('Hello Take Quiz on 8080')
+});
+
+// Title might be cleaner in the URL slug than the quiz ID. Get team input
+// router.get('/:quiz_title/start', (req, res) => {
```

Comment:
my only concerned would be what if there is quizzes with the same title

Reply:
Good point, will keep the unique ID instead-- thank you!